src/index.js

```diff
@@ -374,10 +402,63 @@ let accounts = [];
 let scrollToHandled = false;
 
 const isMetaMaskConnected = () => accounts && accounts.length > 0;
+let isWalletConnectConnected = false;
 
 // TODO: Need to align with @metamask/onboarding
 const isMetaMaskInstalled = () => provider && provider.isMetaMask;
 
+// test id
+const projectId = 'e6360eaee594162688065f1c70c863b7';
+
+const metadata = {
+  name: 'E2e Test Dapp',
+  description: 'This is the E2e Test Dapp',
+  url: 'https://metamask.github.io/test-dapp/',
+  icons: ['https://avatars.mywebsite.com/'],
+};
```

Comment:
metadata that will be seen on the Mobile device, once connected

![image](https://github.com/MetaMask/test-dapp/assets/54408225/a0b44fe6-401f-4630-a70b-46cdd429af8f)